README.md

```
@@ -124,7 +124,15 @@ You can run the tests using:
 
     npm run build && npm run serve
     # In a different terminal:
-    npm run test
+    npm run test:e2e
+
+Unit tests can be run without building the app using:
+
+    npm run test:unit
```

markdownlint-cli2 (0.21.0)

[warning] Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)